server/game/cards/locations/04/towerofthesun.js

```diff
@@ -0,0 +1,20 @@
+const DrawCard = require('../../../drawcard.js');
+
+class TowerOfTheSun extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                onCardPlayed: (event, player, card) => card.getType() === 'event'
```

Comment:
I know, #573 was the reason I did this one. The reason I checked getType() (embarassingly) is that ESLint bugs me if I don't do anything with card. What's the better solution for that? (thanks as usual)